azure-function-http-test/build.gradle

```diff
@@ -1,19 +1,23 @@
+plugins {
+    id 'io.micronaut.build.internal.azure-module'
+}
+
 dependencies {
     annotationProcessor "io.micronaut:micronaut-inject-java"
     api "io.micronaut:micronaut-inject:$micronautVersion"
     api "io.micronaut:micronaut-http-server:$micronautVersion"
     api project(":azure-function-http")
     api "com.microsoft.azure.functions:azure-functions-java-library:$azureFunctionsJavaLibraryVersion"
     api "io.micronaut.servlet:micronaut-servlet-core:$micronautServletVersion"
-    implementation 'org.eclipse.jetty:jetty-server:11.0.2'
+    implementation 'org.eclipse.jetty:jetty-server:9.4.35.v20201120'
     implementation 'jakarta.inject:jakarta.inject-api:2.0.1'
     testCompileOnly "io.micronaut:micronaut-inject-groovy"
     testImplementation "io.micronaut.test:micronaut-test-spock:$micronautTestVersion"
     testImplementation "io.micronaut:micronaut-http-client"
     testAnnotationProcessor "io.micronaut:micronaut-inject-java"
 }
 
-micronautBuild {
+configurations.all {
     resolutionStrategy {
         force "org.eclipse.jetty:jetty-server:9.4.35.v20201120"
```

Comment:
Do you know why this force is required? This is going to work when building locally, but in any case, this isn't propagated to consumers, who would get whatever version is resolved transitively. Do we actually have consumers of `azure-funcions-http-test` (does it need to be published?).

Reply:
well @sdelamo added that in PR https://github.com/micronaut-projects/micronaut-azure/pull/142 that is part of bump of jetty to 11 issue https://github.com/micronaut-projects/micronaut-azure/issues/141 .. but I can just assume that the bump ended up there with this https://github.com/micronaut-projects/micronaut-azure/commit/6abc53fd9b77f050fb9d1e075614ce7d4eba623c and by that fix it was forcefully downgraded ? @sdelamo ?

Reply:
>  Do we actually have consumers of azure-funcions-http-test (does it need to be published?).

yes, it needs to be published. It is jar which we ship so that the users can use to start a server used for testing Azure HTTP functions.

Reply:
I added that snippet to pin the jetty-server to 9.4.35v20201120 and don't udpate to a mayor version uo jetty in a minor verison. 

See: 

https://github.com/micronaut-projects/micronaut-azure/issues/141

I guess we can update it to 11 and release it as part of minor release since this is going to be only in the test classpath.